packages/slate-react/src/plugin/with-react.ts

```diff
@@ -366,6 +374,14 @@ export const withReact = <T extends BaseEditor>(
         onContextChange()
```

Comment:
You need to use this as stated in https://github.com/ianstormtaylor/slate/issues/5521#issuecomment-1762853454

Reply:
I'm not getting what you mean. What should I pass as parameter here?

Reply:
We need 2 new props to `Slate`: `onSelectionChange` and `onValueChange`. These callbacks should be called in `onContextChange` like `onChange`.